.gitignore

```diff
@@ -7,4 +7,8 @@ node_modules
 *.code-*
 *.sublime-*
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+__diff_output__
```

Comment:
I think we should ignore the entire snapshots directory:

```suggestion
__image_snapshots__
```

Otherwise local tests get diffed against the GitHub action's output (or users could potentially push snapshots updated locally that might break GitHub's comparison!) which gives a pretty silly failing diff:

![image](https://user-images.githubusercontent.com/886627/102669084-7c0ff680-4142-11eb-925f-1c01ada68754.png)

But ignoring this folder would actually break diff comparison on the remote, since it would _never_ have anything to compare against, right? So maybe we can get around this by storing the locally generated snapshots in a separate directory, and then just ignore _that?_ That way, GitHub would always diff against remote snapshots!

Reply:
So wrapping `test.browser` in a docker container would let us run the specs in the same environment. So, we should be able to output the same snapshots as on CI.

But yea, we could use a different snapshot dir for CI?

Reply:
> So wrapping test.browser in a docker container would let us run the specs in the same environment. So, we should be able to output the same snapshots as on CI.

And between dev's laptops

Reply:
Again, that sounds much better in the long run, but I'm not sure how long it would take to dockerize this stuff? Maybe we can do that work in a separate PR, once we've got this stuff merged in and running for a little bit? Your call though.

Reply:
I'm tempted to say a couple hours. But the reality is probably a couple days?